2018/python/file_generator.py

```diff
@@ -356,11 +360,15 @@ def validate_quality_edit_file(self, quality_filename):
 			res_df = pd.DataFrame(checker.results)
 			new_df = res_df[(res_df['status']=='failed')]
 
-			#If there are no syntax or validity edits,
-			#the data is written to a new directory for quality 
-			#test files that pass syntax and validity edits. 
+			# The function would ignore TS edits. The edit
```

Comment:
I am confused by these comments, do you mean that the TS row is dropped? Is there a certain condition that causes that to happen?

Reply:
The function ignores TS edits - I can explain more fully.